src/modules/shared/pages/innovation/comments/comments-edit.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+
+    <theme-alert [type]="alert.type" [title]="alert.title" [setFocus]="alert.setFocus">
+      <p> {{ alert.message }} </p>
+    </theme-alert>
+    <theme-spinner *ngIf="pageStatus === 'LOADING'" cssClass="nhsuk-u-margin-9"></theme-spinner>
+    <ng-container *ngIf="pageStatus === 'READY'">
+      <h1 class="nhsuk-u-margin-bottom-3"> {{ pageTitle }} </h1>
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-two-thirds">
+
+          <form [formGroup]="form">
+            <theme-form-textarea controlName="comment" label="Edit your message" [pageUniqueField]="false"
+              lengthLimit="large"></theme-form-textarea>
+            <button typ="submit" [disabled]="form.pristine || !(form.valid && form.dirty)"
```

Comment:
We should be very carefully with the disabled buttons.

They are not recommended by NHS design system: https://service-manual.nhs.uk/design-system/components/buttons

Thoughts?